### PR TITLE
AzFile SDK 0.1.0

### PR DESCRIPTION
**Pull request by mohsha-msft**
- [ ] The purpose of this PR is explained in this or a referenced issue.
- [ ] The PR does not update generated files.
   - These files are managed by the codegen framework at [Azure/autorest.go][].
- [ ] Tests are included and/or updated for code changes.
- [ ] Updates to [CHANGELOG.md][] are included.
- [ ] MIT license headers are included in each file.

[Azure/autorest.go]: https://github.com/Azure/autorest.go
[CHANGELOG.md]: https://github.com/Azure/azure-sdk-for-go/blob/main/CHANGELOG.md

### CODE REVIEW

File: sdk/storage/azfile/autorest.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
+# Code Generation - Azure Blob SDK for Golang
```
Comment (RickWinter) [inline sdk/storage/azfile/autorest.md:1]:
```suggestion
# Code Generation - Azure Storage File SDK for Golang
```

File: sdk/storage/azfile/zz_generated_file_client.go [hunk 2]
```diff
@@ -0,0 +1,1788 @@
...
+		}
+		result.IsServerEncrypted = &isServerEncrypted
+	}
+	if val := resp.Header.Get("x-ms-file-permission-key"); val != "" {
+		result.FilePermissionKey = &val
+	}
+	if val := resp.Header.Get("x-ms-file-attributes"); val != "" {
+		result.FileAttributes = &val
+	}
+	if val := resp.Header.Get("x-ms-file-creation-time"); val != "" {
```
Comment (mohsha-msft) [inline sdk/storage/azfile/zz_generated_file_client.go:454]: Come Here

**Pull request closed without merging**